test/System/String.test.cpp

```diff
@@ -57,6 +57,8 @@ struct StringMaker<sf::U8String>
 };
 } // namespace Catch
 
+// NOLINTBEGIN(cppcoreguidelines-avoid-c-arrays,modernize-avoid-c-arrays)
```

Comment:
Why should it not apply here?

Reply:
Honestly just being a bit lazy. No real excuse for not converting these C arrays as well.